patterns/jolla-hw-adaptation-hammerhead.yaml

```diff
@@ -74,6 +74,11 @@ Requires:
 # Updates after 1.1.7 will autodetect the modem presence during runtime and change
 # layout accordingly
 - jolla-settings-layout
+- voicecall-ui-jolla
```

Comment:
This should be jolla-sailfish-cellular-apps, to pull all needed bits: mms-engine
provisioning-service
simkit
telepathy-ring
voicecall-ui-jolla